Foundation/NSTask.swift

```diff
@@ -7,6 +7,7 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+#if !os(Android) // not available
```

Comment:
Can you tell me the syntax to make specify a NSTask unavailable for android please.